applications/asset_tracker_v2/doc/location_module.rst

```diff
@@ -0,0 +1,117 @@
+.. _asset_tracker_v2_location_module:
+
+Location module
+###############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The location module controls the positioning functionality of the nRF9160 including GNSS and cellular positioning.
+It can be used to retrieve the location of the device in the form of events containing a position, velocity and time (PVT) structure.
+
+
+Features
+********
+
+This section documents the various features implemented by the module.
+
+Location control
+================
+
+The module uses the :ref:`lib_location` to communicate with the nRF9160 modem and control its GNSS and LTE neighbor cell measurement functionalities.
+A location request is started when the module receives an ``APP_EVT_DATA_GET`` event and the ``APP_DATA_LOCATION`` type is listed in the event's ``data_list`` member containing the data types that shall be sampled.
+
+The default configuration for the location request is set as follows:
```

Comment:
If you use active voice, who is the one setting the default configuration?
In reality, it's the location module that we are documenting here.
With active voice, for me it sounds like it would maybe be user setting the default configuration but I'll let you judge that.